09-features.md

```diff
@@ -62,6 +62,8 @@ The origin node:
   * If it requires a feature above, MUST set the corresponding even
     feature bit in all feature fields indicated by the Context column,
     unless indicated that it must set the odd feature bit instead.
+  * If a feature is specified both as an optional feature and as a
+  required feature, the more restrictive option MUST be chosen.
```

Comment:
This is an interpretation rule for other nodes, rather than a requirement on the origin node.  I think it makes sense to move this to the paragraph at the end of this section.

Reply:
Yeah make sense to mee!

Reply:
This section contains requirement for the node that creates a feature bitfield (_writer_). I think it would make more sense here to just say that you MUST NOT set both the optional and mandatory field, it doesn't make sense setting both.

There are no requirements for the _reader_ yet, but you could add one to say that if both the optional and the mandatory feature bits are set, the feature should be treated as mandatory?

Reply:
Done this, thank all